crm_security_group/views/menu_items.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="crm.crm_menu_root" model="ir.ui.menu">
+        <field
+            name="groups_id"
+            eval="[(4, ref('crm_security_group.group_crm_only'))]"
```

Comment:
This file is not correct according specifications. Please check the other comment.